tests/test_unit/test_cdf_tk/test_resource_classes/test_robotics/test_capabilities.py

```diff
@@ -0,0 +1,70 @@
+from collections.abc import Iterable
+from pathlib import Path
+
+import pytest
+
+from cognite_toolkit._cdf_tk.resource_classes.robotics.capabilities import RobotCapabilityYAML
+from cognite_toolkit._cdf_tk.tk_warnings.fileread import ResourceFormatWarning
+from cognite_toolkit._cdf_tk.validation import validate_resource_yaml_pydantic
```

Comment:
generally better to not use external library names in the source code. why can't it be `validate_resource_yaml`?